examples/php-fpm/php/run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+php-fpm -D
+
+screen -d -m sh -c "SCRIPT_FILENAME=./index.php REQUEST_METHOD=GET cgi-fcgi -bind -connect 127.0.0.1:9000"
+
+pid=$(pgrep "php-fpm: master process")
+
+pyroscope connect --pid $pid --spy-name phpspy
```

Comment:
I think this can be simplified by using `exec` instead of connect, no need to separate creating from profiling and the `pgrep` would be needed, right?

Reply:
done